### PR TITLE
Moves lighting SS to last in init order

### PR DESCRIPTION
**Pull request by Cyberboss**
It needs to fire in order for anything to be visible. So after it inits the entire station is dark until initializations are complete.

### CODE REVIEW

**Comment by MrPerson** [comment]
What a bright idea!

File: code/controllers/subsystem/lighting.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@ var/list/lighting_update_overlays  = list() // List of lighting overlays queued
 /datum/subsystem/lighting
 	name = "Lighting"
 	wait = 1
-	init_order = 1
+	init_order = -20
```
Comment (Incoming5643) [inline code/controllers/subsystem/lighting.dm]: -INFINITY

The only way to be absolutely sure

Comment (Cyberboss) [inline code/controllers/subsystem/lighting.dm]: There may be a reason for it to be sub-ceded after. For example, the mapping subsystem used to be INFINITY

Comment (MrPerson) [inline code/controllers/subsystem/lighting.dm]: We could do an entire file of subystem init order defines, if it comes to that

**Pull request merged**